frontend/src/components/housing/submission/SubmissionsNavigator.vue

```diff
@@ -161,6 +164,23 @@ function assignEnquiries() {
   });
 }
 
+// Set multiPermitsNeeded property of each submission to Yes/No (count)
+// if the submission have more than one permit with needed Yes
+function assignMultiPermitsNeeded() {
+  submissions.value.forEach((sub) => {
+    let multiPermitsNeededCount = 0;
+    if (Array.isArray(permits.value)) {
+      permits.value.forEach((permit) => {
+        if (permit.activityId === sub.activityId && permit.needed?.toUpperCase() === BasicResponse.YES.toUpperCase()) {
+          multiPermitsNeededCount++;
+        }
+      });
+    }
```

Comment:
This can be condensed
```
const multiPermitsNeededCount = permits.value.filter(
      (x) => x.activityId === sub.activityId && x.needed?.toUpperCase() === BasicResponse.YES.toUpperCase()
    ).length;
```